api/src/test/java/uk/gov/dhsc/htbhf/claimant/creator/ClaimantLoader.java

```diff
@@ -62,6 +63,9 @@ public void loadClaimantIntoDatabase() throws IOException {
     }
 
     private String createDWPHousehold(ClaimantInfo claimantInfo) {
+        Optional<UCHousehold> existingHousehold = ucHouseholdRepository.findHouseholdByAdultWithNino(claimantInfo.getNino());
```

Comment:
For this test loader, I think it would be better to use `findAllHouseholdsByAdultWithNino` and delete all of the households found rather than presume that there is only ever going to be one and use the `findHouseholdByAdultWithNino` method.

Reply:
agreed - delete all households with that NINO just in case

Reply:
Updated to delete all households.